### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by mfine**
Bring Monad Logger back into the stack. Bring the log lines back.

/cc @mookerji

### CODE REVIEW

File: src/Network/AWS/Flow.hs [hunk 1]
```diff
@@ -36,6 +37,7 @@ import           Safe
 
 register :: MonadFlow m => Plan -> m ()
 register Plan{..} = do
+  logInfoN "event=register\n"
```
Comment (mookerji) [inline src/Network/AWS/Flow.hs]: Hmmm, so I see this here, but not for other flow events (decide, execute, etc.)?

**Comment by mookerji** [comment]
Rest looks fine. Were you planning on logging other stuff or am I missing something?

File: src/Network/AWS/Flow.hs (hunk 1 above)
Comment (mfine) [inline src/Network/AWS/Flow.hs]: Yeah, I'm trying to pick a better formatting library :( Probably going to stick with formatting, as I couldn't find anything reasonable to use `text-format`. So, updated log lines are coming.

**Comment by mfine** [comment]
Yes! Going to bring the rest of the logging back in.

**Comment by mfine** [comment]
Bringing this in.

**Pull request merged**